### PR TITLE
Add gpuType to pai mode

### PR DESCRIPTION
**Pull request by demianzhang**

### CODE REVIEW

File: docs/en_US/PaiMode.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ Compared with LocalMode and [RemoteMachineMode](RemoteMachineMode.md), trial con
     * Optional key. Set the virtualCluster of OpenPAI. If omitted, the job will run on default virtual cluster.
 * shmMB
     * Optional key. Set the shmMB configuration of OpenPAI, it set the shared memory for one task in the task role.
+* gpuType
+    * Optional key. Specify the GPU type to be used in the tasks. If omitted, the job will run on any gpu type
```
Comment (xuehui1991) [inline docs/en_US/PaiMode.md]: Can we list some optional choice here...

Comment (demianzhang) [reply docs/en_US/PaiMode.md]: Sure. However, nni 179 cluster only use K80 gpuType, according to [pai issue](https://github.com/microsoft/pai/blob/master/docs/hadoop-ai/README.md#known-issues) gpuType will work after manually updating the configuration on PAI.

**Comment by fanyangCS** [comment]
please do not add GpuType support, we are going to obsolete the field.

**Comment by leckie-chn** [comment]
@scarlett2018

**Comment by leckie-chn** [comment]
close this PR due to deprecation of `gpuType` in PAI

**Pull request closed without merging**